lib/developer_portal/app/controllers/developer_portal/admin/account/personal_details_controller.rb

```diff
@@ -50,6 +46,7 @@ def deny_unless_can_update
   end
 
   def redirect_path
+    #TODO: Test this
```

Comment:
OK, I think that this code is quite outdated...

I am not sure, but I think in the past (like, long long time ago), the admin portal (provider) users could log in to the developer portal. Now it's not the case, in fact the admin portal and developer portal are two different Rails engines (on different domains also), so there is no way to jump between one and another. Also, the users of the two portals are different, provider users can't login into the developer portal.

Also, I think `arams[:origin] == "users"` does not apply here.

In the admin portal it is valid, because there are two places from where you can go to the personal details update form:
1. From the menu **Account Settings > Personal > Personal Details** https://github.com/3scale/porta/blob/3017562449d3275840cae6afe338a01c1dcca899/app/views/shared/provider/navigation/account/_personal.html.slim#L4
2. From the list of the provider users **Account Settings > Users > Listing** 
https://github.com/3scale/porta/blob/3017562449d3275840cae6afe338a01c1dcca899/app/views/provider/admin/account/users/index.html.slim#L27
Which is where it may have `params[:origin] == "users"`

In the developer portal though there is just a single entrypoint, and `origin=users` is not set anywhere.
So this can be simplified by always redirecting to the same path.
Now, there are two options: either stay on the Personal Details update form (`admin_account_personal_details_path`), or to go back to the users list page (`admin_account_users_path`). I am not sure which is the best here.

Reply:
I would say, from an UI perspective, that it makes more sense going to the users list page (`admin_account_users_path`), but I'm open to feedback.